packages/backend/src/Application.ts

```diff
@@ -201,8 +201,10 @@ export class Application {
       await databaseService.migrateToLatest()
 
       await apiServer.listen()
-      await syncScheduler.start()
-      await blockDownloader.start()
+      if (config.enableSync) {
```

Comment:
What is the reason for introducing this?

Reply:
I wanted to connect to production database locally but I also didn't want to modify it

Reply:
I can extract this to a separate PR if you want

Reply:
No, that's fine to be here - simple enough to understand